src/SIM/SIMbase.C

```diff
@@ -1617,6 +1617,7 @@ bool SIMbase::systemModes (std::vector<Mode>& solution,
   for (int i = 1; i <= nev && ok; i++)
   {
     solution[i-1].eigNo = i;
+    solution[i-1].eqnVec = eigVec.getColumn(i);
     if (!mySam->expandVector(eigVec.getColumn(i),solution[i-1].eigVec))
```

Comment:
sub-sub optimizing: Could then use `solution[i-1].eqnVec` in the `expandVector()` call to avoid calling `getColumn()` twice

Reply:
yup yup. done.